Week-4/Exercise-4.2/prototype.js

```diff
@@ -0,0 +1,16 @@
+const Person = function () {};
+
+Person.prototype.initialize = function (name, age) {
+  this.name = name;
+  this.age = age;
+  this.teach = function(subject) {
+      console.log(this.name + " is teaching " + subject);
+      console.log(this.name + " is of age " + this.age);
```

Comment:
Person prototype will stay as it is . Teacher should be newly created and should inherit name of person prototype and will have subject as new parameter